### PR TITLE
Increase upgrade retry health check timeout to fix upgrade CI/CD job failing

### PR DESCRIPTION
**Pull request by AmeyShikerkar**
Signed-off-by: Amey Shikerkar <amey.shikerkar@ibm.com>

### CODE REVIEW

**Review by bpradipt — DISMISSED** [review]
/lgtm

File: playbooks/roles/ocp-upgrade/tasks/main.yaml [hunk 1]
```diff
@@ -1,22 +1,16 @@
...
-  until: "'AVAILABLE' in oplog.stdout"
-  retries: 3
-  delay: 3
-
-- name: Process cluster operators details
-  shell: |
-    echo "{{ oplog.stdout }}" | awk '/PROGRESSING/,EOF {print $4}' | grep 'True' | wc -l
-    echo "{{ oplog.stdout }}" | awk '/DEGRADED/,EOF {print $5}' | grep 'True' | wc -l
-  register: opcount
+- name: Retry till all co are Available/NOT PROGRESSING/NOT DEGRADED state
```
Comment (yussufsh) [inline playbooks/roles/ocp-upgrade/tasks/main.yaml]: Can we have either blocks or title format for all?
```
Available/NOT Progressing/NOT Degraded
```

Comment (AmeyShikerkar) [reply playbooks/roles/ocp-upgrade/tasks/main.yaml]: Updated title to 'Check all co are in 'Available' state and not in 'Progressing' or 'Degraded' state'

**Review by yussufsh — APPROVED** [review]
/lgtm

**Review by bpradipt — APPROVED** [review]
/lgtm

**Pull request merged**